src/cpp/utils/threading/threading_win32.ipp

```diff
@@ -70,6 +82,7 @@ static void configure_current_thread_affinity(
 void apply_thread_settings_to_current_thread(
         const fastdds::rtps::ThreadSettings& settings)
 {
+    configure_current_thread_priority(settings.priority);
```

Comment:
If we cannot change sched policy in windows, he should state so in ThreadSettings.hpp

Reply:
2f7faec